components/dashboard/src/projects/NewProject.tsx

```diff
@@ -229,54 +259,73 @@ export default function NewProject() {
+                            <img src={NoAccess} title="No Access" className="m-auto mb-4" />
+                            <h3 className="mb-2 text-gray-600 dark:text-gray-400">
+                                No Access
+                            </h3>
+                            <span className="dark:text-gray-500">
+                                Authorize GitHub (github.com) or select a different account.
+                            </span>
+                            <br />
+                            <button className="mt-6" onClick={() => reconfigure()}>Authorize</button>
+                        </div>
+                    </div>)}
+                </div>
+
+            </div>
+            {isGitHub() && (
+                    <div className="pt-3 bg-gray-100 rounded-b-xl border-gray-100 border">
+                        <div className="text-gray-500 text-center w-96 mx-8">
+                            Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
+                        </div>
+                        {isGitHub() && noOrgs && (
+                            <div className="text-gray-500 mx-auto text-center">
+                                Missing organizations? <a href="javascript:void(0)" onClick={e => grantReadOrgPermissions()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Grant permissions</a>
                             </div>
-                        ))}
+                        )}
                     </div>
-                )}
-                {reposToRender.length === 0 && (
-                    <p className="text-center ">not found</p>
-                )}
-            </div>
-
-            <div className="px-3 pt-3 bg-gray-100">
-                <div className="text-gray-500 text-center">
-                    Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
-                </div>
-                {isGitHub() && noOrgs && (
-                    <div className="text-gray-500 mx-auto text-center">
-                        Missing organizations? <a href="javascript:void(0)" onClick={e => grantReadOrgPermissions()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Grant permissions</a>
+            )}
+        </>
+        );
+
+        const renderEmptyState = () => (<div>
+            <div className="mt-8 border rounded-xl border-gray-100 dark:border-gray-700 flex-col">
+                <div>
+                    <div className="px-12 py-16 text-center text-gray-500 bg-gray-50 dark:bg-gray-800 rounded-xl">
+                        <h3 className="mb-2 text-gray-600 dark:text-gray-400">
```

Comment:
**nitpick**: Minor color shift and small hack for improving the user experience while loading time, until we have a better loading indicator like the spinner described in https://github.com/gitpod-io/gitpod/issues/4948#issuecomment-895927769.

```suggestion
                        <h3 className="mb-2 text-gray-400 dark:text-gray-600 animate-pulse">
```